File: examples/js/get__list.js

const authToken = process.env.AIOPS_TOKEN || 'YOUR_TOKEN';
// 示例：GET /list
fetch(`http://localhost:8000/api/v1/list`, {
    method: 'GET',
    headers: { Authorization: `Bearer ${authToken}` }
})
.then(res => res.json())
.then(data => console.log(data))
.catch(err => console.error('Request error:', err));

File: examples/python/get__list.py
# -*- coding: utf-8 -*-
import requests

# 示例：GET /list
# 使用 GET 方法请求 /list
url = "http://localhost:8080/api/v1/list"
try:
    response = requests.get(url)
    print("Status:", response.status_code)
    print("Response:", response.json())
except Exception as e:
    print("Request failed:", e)
